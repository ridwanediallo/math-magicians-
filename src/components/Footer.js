import React from 'react';
import './Navbar.css';

const Footer = () => (
  <footer className="footer">
    <h2 className="copyright">©️ RidwanApps 2022</h2>
  </footer>
);

export default Footer;
