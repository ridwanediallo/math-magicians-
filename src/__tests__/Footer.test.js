import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

it('Render Footer correctly', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
