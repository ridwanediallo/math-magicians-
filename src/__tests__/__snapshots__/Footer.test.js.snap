// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Footer correctly 1`] = `
<footer
  className="footer"
>
  <h2
    className="copyright"
  >
    ©️ RidwanApps 2022
  </h2>
</footer>
`;
